### PR TITLE
[139] Product Page using HTML

### PR DESCRIPTION
**Pull request by HalimahOladimeji**
Screenshot 2020-11-20 122208
Screenshot 2020-11-20 122232

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
@HalimahOladimeji This task requires only HTML, I can see some CSS styling in your file. Kindly follow the instructions in the task criteria.

Also, note that when you get to the CSS task, you should have a separate file for your styling. Internal styling is not recommended.

**Comment by HalimahOladimeji** [comment]
I have made the changes. Kindly review it. Thanks.

File: submissions/HTML/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+</div>
+
+<div style="padding-left:16px">
+  <h1>PREMIUM TECHNOLOGIES</h1>
+</div>
+<div class="row">
+    <div class="column">
+      <img src="images/zflip.jpg" alt="zflip" style="width:70%">
+    </div>
+    <div class="column">
```
Comment (Ireti203) [inline submissions/HTML/index.html:23]: The entire file is not properly formatted. Kindly correct this.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by Ireti203** [comment]
Attach an image of your webpage after removing the styling

**Comment by HalimahOladimeji** [comment]
I have made the changes. I've attached the screenshots here. Thanks

File: submissions/HTML/index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+
```
Comment (Ireti203) [inline submissions/HTML/index.html:4]: Remove whitespace

File: submissions/HTML/index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+<body>
+
```
Comment (Ireti203) [inline submissions/HTML/index.html:6]: Remove whitespace

File: submissions/HTML/index.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+<div class="topnav">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">News</a>
+  <a href="#contact">Contact</a>
+  <a href="#about">About</a>
+</div>
```
Comment (Ireti203) [inline submissions/HTML/index.html:12]: The entire file isnt properly formatted. This div is under the body tag, they shouldnt start at the same point. Kindly check the samples in the lesson video/slides.

File: submissions/HTML/index.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                <li><a href="https://www.twitter.com">Twitter</a></li>
+                <li><a href="https://www.linkedin.com">LinkedIn</a></li>
+                <li><a href="https://www.medium.com">Medium</a></li>
+            </ul>
+        </p>
+        <p &copy; PremiumTechnologies</p>
+    </div>
+</footer>
+</body>
+</html>
```
Comment (Ireti203) [inline submissions/HTML/index.html:82]: Kindly format properly. It is good practice to write clean and readable code.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by HalimahOladimeji** [comment]
I have made the changes. Find attached.

**Comment by Ireti203** [comment]
Kindly change the screenshot attached to this PR to what you currently have. The one you uploaded initially has CSS, kindly remove it and update.

**Comment by HalimahOladimeji** [comment]
I have done that. Kindly check it. Thanks.

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**